web/src/api/api.ts

```diff
@@ -231,7 +231,12 @@ export async function loadFragments(pair: Pair, kmers: ObjMap<Kgram>, customOpti
     [fileToTokenizedFile(pair.leftFile), fileToTokenizedFile(pair.rightFile)]
   );
   const reportPair = report.scoredPairs[0].pair;
-  pair.fragments = parseFragments(reportPair.fragments(), kmers);
+  const hashToKmers: ObjMap<Kgram> = {};
```

Comment:
I generally prefer using [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), but maybe we should have a unified style for this?